helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -19,19 +19,17 @@
  * under the License.
  */
 
+import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Maps;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
 import org.apache.helix.HelixException;
 import org.apache.helix.HelixProperty;
 import org.apache.helix.ZNRecord;
 import org.apache.helix.api.config.HelixConfigProperty;
 import org.apache.helix.api.config.StateTransitionThrottleConfig;
 import org.apache.helix.api.config.StateTransitionTimeoutConfig;
 
+import java.util.*;
```

Comment:
Is there any reason you import all classes in util instead of importing specific classes you need? In my opinion, I would prefer to import specific classes I need to avoid conflicts.

Reply:
I think the default limitation in Intellij to merge the imports is 5. I guess we need a better format file.
I will fix this PR later.

Reply:
Not sure if it is a Helix rule. But I guess this count could be set in Intellij: Preferences (⌘ + , on macOS / Ctrl + Alt + S on Windows) > Editor > Code Style > Java > Imports tab ->  Names count to use static import with '*'. By default it is 5.